backend/src/test/java/com/yagubogu/auth/MemberIntegrationTest.java

```diff
@@ -0,0 +1,62 @@
+package com.yagubogu.auth;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import com.yagubogu.auth.client.AuthGateway;
+import com.yagubogu.auth.dto.AuthResponse;
+import com.yagubogu.auth.dto.LoginRequest;
+import com.yagubogu.auth.dto.LoginResponse;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.time.Instant;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.annotation.DirtiesContext.ClassMode;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.bean.override.mockito.MockitoBean;
+
+@TestPropertySource(properties = {
+        "spring.sql.init.data-locations=classpath:test-data.sql"
+})
+@DirtiesContext(classMode = ClassMode.AFTER_EACH_TEST_METHOD)
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+public class MemberIntegrationTest {
+
+    @LocalServerPort
+    private int port;
+
+    @MockitoBean
+    private AuthGateway authGateway;
+
+    @BeforeEach
+    void setUp() {
+        RestAssured.port = port;
+    }
+
+    @DisplayName("로그인을 수행한다")
+    @Test
+    void login() {
+        // given
+        AuthResponse authResponse = new AuthResponse("accounts.google.com", "sub-test-unique-01", "azp",
+                "this-is-client-id",
+                111L, Instant.now().plusSeconds(3000).getEpochSecond(), "email", true, "name",
+                "picture", "givenName", "familyName", "ko");
+        when(authGateway.validateToken(any())).thenReturn(authResponse);
+
+        // when
+        RestAssured.given().log().all()
+                .contentType(ContentType.JSON)
+                .body(new LoginRequest("ID_TOKEN"))
+                .when().post("/api/auth/login")
+                .then().log().all()
+                .statusCode(200)
+                .extract()
+                .as(LoginResponse.class);
```

Comment:
R: 결과를 꺼내기만 하고, 실제 검증은 없네요!
response가 잘 받아와졌는지 확인하면 좋을 것 같아요

```suggestion
assertThat(response.getMemberId()).isNotNull();
assertThat(response.getAccessToken()).isNotBlank()
```

Reply:
넵! 검증까지 추가하겠습니다 :)